jerry-core/include/jerryscript-compiler.h

```diff
@@ -41,6 +40,7 @@ extern "C"
 #define JERRY_ATTR_PURE __attribute__((pure))
 #define JERRY_ATTR_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
 
+#define JERRY_ALWAYS_INLINE inline __attribute__((always_inline))
```

Comment:
Please add a note about the usage of this macro: 
 - before it's usage `static` or `extern` modifier must be used.

Reply:
done in latter line